### PR TITLE
fix(RequestsPortlet): Unable to reopen CR, Open Components to display open releases, clearing progress to show percentage

### PR DESCRIPTION
**Pull request by afsahsyeda**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

**Description**

1. Requesting User and Clearing Expert can reopen a CR.
2. Requesting User can now close the CR.
3. Open Components has been renamed to Open Releases and displays count of Releases in NEW state.
4. Clearing Progress is now presented in percentage.
image

Closes: #2181

### CODE REVIEW

**Comment by akshitjoshii** [comment]
Testing completed, working as expected.
pr_2182
pr_2182_01

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/moderation/ModerationPortletUtils.java [hunk 1]
```diff
@@ -128,21 +129,28 @@ public static AddDocumentRequestSummary updateClearingRequest(PortletRequest req
...
                         log.warn("Invalid agreed clearing date: " + agreedDate + " is entered, by user: "+ user.getEmail());
                         return requestSummary.setMessage("Invalid agreed clearing date");
                     }
+                    clearingRequest.setClearingState(requestState);
                     clearingRequest.setAgreedClearingDate(CommonUtils.nullToEmptyString(agreedDate));
-                    clearingRequest.setClearingState(ClearingRequestState.findByValue(parseInt(status)));
                     clearingRequest.setPriority(ClearingRequestPriority.findByValue(parseInt(priority)));
                     clearingRequest.setClearingType(ClearingRequestType.findByValue(parseInt(clearingType)));
                 }
+                if (requestingUser.equals(user.getEmail())) {
```
Comment (hoangnt2) [inline frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/moderation/ModerationPortletUtils.java:149]: I think in this case, if the requestingUser has role CLEARING_EXPERT and the clearing state is CLOSED, the clearing request state will be set twice time.
Maybe you should replace **if** to **else if**

**Review by hoangnt2 — CHANGES_REQUESTED** [review]

**Comment by afsahsyeda** [comment]
Hi @hoangnt2 ,

I have addressed the review comments. Please check.

**Review by hoangnt2 — COMMENTED** [review]
Change look good to me

**Review by hoangnt2 — APPROVED** [review]

**Pull request merged**